kratos/utilities/rve_periodicity_utility.h

```diff
@@ -193,6 +194,14 @@ class KRATOS_API(KRATOS_CORE) RVEPeriodicityUtility
         const Vector& rTranslationVector
         );
 
+    template<std::size_t TDim>
```

Comment:
Doxygen?

Reply:
I don't think it is needed in this case as it is a `private` function to be specialized. Doxygen is already present in the `AssignPeriodicity`.